addons/mail/static/src/models/chat_window/chat_window.js

```diff
@@ -216,7 +216,12 @@ function factory(dependencies) {
              * @returns {integer}
              */
             const _getNextIndex = index => {
-                const directionOffset = reverse ? -1 : 1;
+                let directionOffset;
+                if (this.env.messaging.locale.textDirection === 'ltr') {
+                    directionOffset = reverse ? 1 : -1;
+                } else {
+                    directionOffset = reverse ? -1 : 1;
+                }
```

Comment:
After you rebase on my fix for rtl (it is being merged now), I would like to test this again before we merge the PR.

Reply:
It's working for me

Reply:
It's working but not in the right order as I suspected since the start. Both are going from left to right with this code.

When using rtl, it must go from right to left when using tab, so there shouldn't be any particular code for it here.

The code displaying them in reverse order based on their index is naturally making it so that the offset change when cycling has to be the same in both cases.

Reply:
I think cycling to next moves the same way to next visible chat window, regardless of RLT or LTR. This seems to be solely based on visible order index.